packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php

```diff
@@ -204,11 +204,23 @@ class="text-gray-600 dark:text-gray-300"
                                     </p>
 
 
-                                    <p
-                                        class="text-gray-600 dark:text-gray-300"
-                                        v-text="data.status"
-                                    >
-                                    </p>
+                                    <template v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.paid')'">
+                                        <p class="label-active">
+                                            @lang('admin::app.sales.transactions.index.datagrid.paid')
+                                        </p>
+                                    </template>
+
+                                    <template v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.pending')'">
+                                        <p class="label-pending">
+                                            @lang('admin::app.sales.transactions.index.datagrid.pending')
+                                        </p>
+                                    </template>
+
+                                    <template v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.completed')'">
+                                        <p class="label-completed">
+                                            @lang('admin::app.sales.transactions.index.datagrid.completed')
+                                        </p>
+                                    </template>
```

Comment:
We can use `v-else-if` here to optimize the code slightly, and the v-if condition can be added directly to the `<p>` tag for better readability and efficiency.